rules/add-useful-and-concise-figure-captions/rule.md

```diff
@@ -108,6 +108,7 @@ This helps:
 
 ::: info
 **Note:** The exception is for promotional videos where the caption may undesirably impact the look and feel of your page.
```

Comment:
Just need a \ so the line break happens in the correct place, unless the intent is just to keep the sentence going on the same line

```suggestion
**Note:** The exception is for promotional videos where the caption may undesirably impact the look and feel of your page.\
```